rts/Lua/LuaHandleSynced.cpp

```diff
@@ -882,31 +879,26 @@ bool CSyncedLuaHandle::AllowUnitTransportLoad(
 	const float3& loadPos,
 	bool allowed
 ) {
-	RECOIL_DETAILED_TRACY_ZONE;
 	LUA_CALL_IN_CHECK(L, true);
-	luaL_checkstack(L, 2 + 9, __func__);
+	luaL_checkstack(L, 6, __func__);
 
 	static const LuaHashString cmdStr(__func__);
-
-	// use engine default if callin does not exist
 	if (!cmdStr.GetGlobalFunc(L))
-		return allowed;
+		return true; // the call is not defined
 
 	lua_pushnumber(L, transporter->id);
-	lua_pushnumber(L, transporter->unitDef->id);
```

Comment:
what's with the random changes?

Reply:
Was just simplifying while I was in here. Was going to review all call sites for AllowUnitTransfer and test them anyway. These served an important function? I will revert these if you think they were necessary or maybe helpful downstream.

Reply:
Will revert/review this. Was attempting to simplify since I was going to test the downstream anyway.